substrate/frame/bounties/src/lib.rs

```diff
@@ -1009,6 +1045,55 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		Ok(())
 	}
+
+	/// Helper function to calculate the bounty storage deposit.
+	fn calculate_bounty_deposit(
+		description: &BoundedVec<u8, T::MaximumReasonLength>,
+	) -> BalanceOf<T, I> {
+		T::BountyDepositBase::get() +
+			T::DataDepositPerByte::get() * (description.len() as u32).into()
```

Comment:
Saturating math could help, otherwise someone can configure their runtime to overflow here.